defaults/main.yml

```diff
@@ -1,13 +1,13 @@
 ---
 # The directory where the Terraform and terraform-docs binaries are to
 # be installed
-install_dir: /usr/local/bin
+terraform_install_dir: /usr/local/bin
 # The group of the Terraform and terraform-docs binaries
-group: root
+terraform_group: root
 # The mode of the Terraform and terraform-docs binaries
```

Comment:
The order of these two variables should be reversed to keep them sorted alphabetically.

Reply:
Addressed in [09c15c2](https://github.com/cisagov/ansible-role-terraform/pull/24/commits/09c15c20497c773232ce68946159ed43ba60b386) - also moved terraform_docs_version